### DISCUSSION TITLE
My version of Min

### DISCUSSION THREAD

**Discussion by CypherPotato** [minbrowser/min#2333] [Show and tell]
Hello!

I liked the project. Thank you for creating Min.

I made a customized version of Min, catered to my tastes. It involves changes to CSS and some modifications to Javascript.

Style inspired by Safari 15.

![image](https://github.com/minbrowser/min/assets/17441745/e527b9f8-b8d3-49cb-a3b9-2bd11c090246)
![image](https://github.com/minbrowser/min/assets/17441745/bb20a4f6-aabc-4181-8fed-9c2cafbed234)
![image](https://github.com/minbrowser/min/assets/17441745/8da4d941-ff4f-42bb-962d-e4d9f2df61c3)

I also intended to make some changes, but I haven't had time yet. They include:

- Show the tab favicon.
- Have support for multiple profiles. To do this, I intended to make a simple tool in C# to clone Min in different directories or point only to the user-data folder, but I didn't dig into the code to understand how the information is stored.
- Sync data. To do this, I would synchronize primarily on a private server, but I haven't tested it yet. I need to test.

**Comment by CypherPotato** [comment]
Source code available at https://github.com/CypherPotato/min, at commit https://github.com/CypherPotato/min/commit/1c236e19728862984e13f7bfc101284c9b2066b7

**Comment by PalmerAL** [comment]
This is really cool, thanks for sharing!

I think we should consider a floating search bar like in your screenshot. I'm less sure about the tabs - it seems it would be harder to see your tabs when you have many open. But I'd have to try it and find out.

> Have support for multiple profiles. To do this, I intended to make a simple tool in C# to clone Min in different directories or point only to the user-data folder, but I didn't dig into the code to understand how the information is stored.

We can do this by calling app.setPath('userData') / app.setPath('sessionData') on startup: https://www.electronjs.org/docs/latest/api/app#appsetpathname-path . Most of the work is building a UI to support managing the profiles.

**Reply by CypherPotato** [reply]
Are you already working on supporting multiple profiles? Maybe I can help.

**Reply by PalmerAL** [reply]
I haven't done any work on it, but if you want to that would be great.

**Reply by Tedd-Codes** [reply]
the ui looks great

**Comment by ioc2e3** [comment]
Sorry, I can't find the place to ask...

Why is there no more webkit-based browser on Windows?

**Reply by Tedd-Codes** [reply]
idk

**Reply by Tedd-Codes** [reply]
could be a cool idea

**Comment by 097115** [comment]
@CypherPotato, looks great, thanks for sharing!

I also wonder... is it possible to completely hide the tabbar? Via some CSS adjustments, maybe? Would be really grateful for a hint, if possible :)

**Reply by PalmerAL** [reply]
You'll want to edit tabBar.css to have `#navbar { display: none }` and then edit navbarHeight in getViewBounds in webviews.js so that the tab content resizes appropriately. If you enable separate titlebar mode, it might be usable after that, although you'll probably find some other UI issues.

**Reply by 097115** [reply]
Thanks a lot, @PalmerAL !
